controller/konnect/reconciler_konnectapiauth.go

```diff
@@ -244,3 +250,16 @@ func getTokenFromKonnectAPIAuthConfiguration(
 
 	return "", fmt.Errorf("unknown KonnectAPIAuthType: %s", apiAuth.Spec.Type)
 }
+
+func getKonnectServerURL(serverURL string) (string, error) {
+	match, err := regexp.MatchString(`.*://`, serverURL)
```

Comment:
Nit: we might want to compile the regex once in global var like 
```go
var serverURLRegex = regexp.MustCompile(`.*://`) 
```
and use it here to avoid compiling it on every call.

Reply:
done in e5e2ff580c8cba45f20bee5568ec573149fd1c6b